runtime/common/ServerHelper.h

```diff
@@ -103,6 +104,19 @@ class ServerHelper : public registry::RegisteredType<ServerHelper> {
   virtual ServerJobPayload
   createJob(std::vector<KernelExecution> &circuitCodes) = 0;
 
+  /// @brief Create a job payload for the provided TBI parameters
+  virtual ServerJobPayload createJob(cudaq::orca::TBIParameters params) {
+    std::vector<ServerMessage> jobs;
+    ServerMessage job;
+    jobs.push_back(job);
+
+    std::map<std::string, std::string> headers;
+
+    // Return a tuple containing the job path, headers, and the job message
+    auto ret = std::make_tuple("", headers, jobs);
+    return ret;
```

Comment:
I think it makes more sense to throw an exception in this case rather than returning a dummy payload like this.

Reply:
Throwing exception